### PR TITLE
Rebuild the openstack features page for ua4i

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Rebuild the openstack features page for ua4i

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /openstack/features
- Check it matches [the copydoc](https://docs.google.com/document/d/1odFXuFtN1GjuZvgd8tW48HB4J9PjZJtP3w6wY0xlmog/edit#)

## Issue / Card
Fixes https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5018

### CODE REVIEW

File: templates/openstack/features.html [hunk 1]
```diff
@@ -246,12 +242,33 @@ <h3>Kubernetes support included</h3>
...
       <div class="col-7">
         <h2>Storage</h2>
+        <p>Storage performance and economics are tricky to balance in a cloud environment. Canonical will help you architect your storage across the cluster to balance price and performance, ensuring the right mix of resilience, latency, IOPS and integrity for your particular deployment.</p>
+      </div>
+    </div>
+    <div class="row p-divider">
+      <div class="col-6 p-divider__block">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/4f1e16bf-image-swift.svg">
```
Comment (pmahnke) [inline templates/openstack/features.html]: {{ ASSET_SERVER_URL }}

File: templates/openstack/features.html [hunk 2]
```diff
@@ -246,12 +242,33 @@ <h3>Kubernetes support included</h3>
...
+            <h3 class="p-heading-icon__title">Swift</h3>
+          </div>
+          <p>The high performance S3-compatible object store developed and shipped with OpenStack, ideal for storing unstructured data.</p>
+        </div>
+      </div>
+      <div class="col-6 p-divider__block">
+        <div class="p-heading-icon">
+          <div class="p-heading-icon__header">
+            <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/636b94fd-ceph.svg">
```
Comment (pmahnke) [inline templates/openstack/features.html]: {{ ASSET_SERVER_URL }}

File: templates/openstack/features.html [hunk 3]
```diff
@@ -246,12 +242,33 @@ <h3>Kubernetes support included</h3>
     <div class="row">
```
Comment (pmahnke) [inline templates/openstack/features.html]: I don't like this section's look

can we try this?

```
<section class="p-strip is-bordered">
    <div class="row">
      <h2>Storage</h2>
    </div>
        <div class="row">
        <div class="col-6">
        <p>Storage performance and economics are tricky to balance in a cloud environment. Canonical will help you architect your storage across the cluster to balance price and performance, ensuring the right mix of resilience, latency, IOPS and integrity for your particular deployment.</p>
        <ul class="p-list">
          <li class="p-list__item is-ticked">12TB usable storage per node included</li>
          <li class="p-list__item is-ticked">Mix of local direct-attach and network storage</li>
          <li class="p-list__item is-ticked">Distributed software-defined storage vs SAN assessment</li>
          <li class="p-list__item is-ticked">Performance tuning for local and network storage</li>
          <li class="p-list__item is-ticked">Understanding storage compression</li>
          <li class="p-list__item is-ticked">Understanding storage resilience</li>
          <li class="p-list__item is-ticked">Third-party storage solutions available for integration</li>
          <li class="p-list__item is-ticked">Object stores and other higher-level storage constructs</li>
          <li class="p-list__item is-ticked">Hierarchical storage configurations</li>
        </ul>
        <p><a href="/openstack/storage" class="">Learn more about storage with Ubuntu Advantage for Infrastructure ›</a></p>
    </div>
        <div class="col-6">
        <div class="p-heading-icon">
          <div class="p-heading-icon__header">
            <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}4f1e16bf-image-swift.svg">
            <h3 class="p-heading-icon__title">Swift</h3>
          </div>
          <p>The high performance S3-compatible object store developed and shipped with OpenStack, ideal for storing unstructured data.</p>
        </div>
          <div class="p-heading-icon">
          <div class="p-heading-icon__header">
            <img class="p-heading-icon__img" src="{{ ASSET_SERVER_URL }}636b94fd-ceph.svg">
            <h3 class="p-heading-icon__title">Ceph</h3>
          </div>
          <p>A converged storage framework that has been designed to present object, block, and file storage from a single distributed computer cluster.</p>
        </div>
      </div>
  </div>
</section>
```

image

**Comment by anthonydillon** [comment]
@pmahnke all done, thanks

**Comment by anasereijo** [comment]
- [ ] on storage section can we try put Swift and Ceph into Header cards?  `p-card__thumbnail`

- [ ] the hero section is white and it's correct. But at the moment when we open this page, it looks very white. I suggest making the 2nd section light grey and alternate with white through the rest of the page. (basically swapping what we have now)

**Comment by anthonydillon** [comment]
@anasereijo done the changes you suggested.

**Comment by anasereijo** [comment]
@anthonydillon  looks much better with the p-cards! 👍  can you make them 5 col instead of 6? Looks a bit wide now. Thanks

**Comment by anasereijo** [comment]
@anthonydillon much better! thanks for making the changes.

**Pull request merged**